plasmapy/__init__.py

```diff
@@ -65,11 +66,11 @@
 except ImportError:
     raise ImportError("Unable to load PlasmaPy subpackages.")
 
-# Allow astropy.units to be imported from PlasmaPy. This is the
-# only place in the code where units should not be abbreviated as u.
+# Allow astropy.units to be imported as plasmapy.units. This is the only
+# place in the code where units should not be abbreviated as u.
```

Comment:
... was that in here all along? Wow.

What's even neater is that this is literally the only place on the entire Internet where `plasmapy.units` is mentioned. :grinning:

Reply:
This probably means that we should add some documentation on this then!

Feel free to also do a web search on "puns about computational magnetohydrodynamics".

Reply:
I've been wondering about that too.  My hesitation is that changing the imports to `plasmapy.units` would be that it obscures that the subpackage is ultimately being imported from `astropy.units`.

Reply:
Yeah. I really wouldn't mind getting rid of this entire `units` import, as the only thing it really seems to do is pollute the namespace. This is a subset of the output of `vars(plasmapy)`:

```python
{
'__name__': 'plasmapy',
 '__doc__': 'A community-developed and community-driven open source core Python package for plasma physics.',
 '__package__': 'plasmapy',
 '__loader__': <_frozen_importlib_external.SourceFileLoader at 0x7f8f78658a58>,
 '__spec__': ModuleSpec(name='plasmapy', loader=<_frozen_importlib_external.SourceFileLoader object at 0x7f8f78658a58>, origin='/home/dominik/Code/PlasmaPy/plasmapy/__init__.py', submodule_search_locations=['/home/dominik/Code/PlasmaPy/plasmapy']),
 '__path__': ['/home/dominik/Code/PlasmaPy/plasmapy'],
 '__file__': '/home/dominik/Code/PlasmaPy/plasmapy/__init__.py',
 '__cached__': '/home/dominik/Code/PlasmaPy/plasmapy/__pycache__/__init__.cpython-36.pyc',
# ...
 'sys': <module 'sys' (built-in)>,
 'constants': <module 'plasmapy.constants' from '/home/dominik/Code/PlasmaPy/plasmapy/constants/__init__.py'>,
 'utils': <module 'plasmapy.utils' from '/home/dominik/Code/PlasmaPy/plasmapy/utils/__init__.py'>,
 '_split_version': <function plasmapy._split_version(version)>,
 '_compiler': <module 'plasmapy._compiler' from '/home/dominik/Code/PlasmaPy/plasmapy/_compiler.cpython-36m-x86_64-linux-gnu.so'>,
 'cython_version': <module 'plasmapy.cython_version' from '/home/dominik/Code/PlasmaPy/plasmapy/cython_version.py'>,
 'version': <module 'plasmapy.version' from '/home/dominik/Code/PlasmaPy/plasmapy/version.py'>,
 '__version__': '0.1.0dev01709.dev1709',
 'atomic': <module 'plasmapy.atomic' from '/home/dominik/Code/PlasmaPy/plasmapy/atomic/__init__.py'>,
 'mathematics': <module 'plasmapy.mathematics' from '/home/dominik/Code/PlasmaPy/plasmapy/mathematics/__init__.py'>,
 'physics': <module 'plasmapy.physics' from '/home/dominik/Code/PlasmaPy/plasmapy/physics/__init__.py'>,
 'classes': <module 'plasmapy.classes' from '/home/dominik/Code/PlasmaPy/plasmapy/classes/__init__.py'>,
 'diagnostics': <module 'plasmapy.diagnostics' from '/home/dominik/Code/PlasmaPy/plasmapy/diagnostics/__init__.py'>,
 'units': <module 'astropy.units' from '/home/dominik/.anaconda3/envs/plasmapy/lib/python3.6/site-packages/astropy/units/__init__.py'>}
```
This is basically what ipython's autocomplete and other packages of this sort deal with. The probably unnecessary ones here are `units` (as it kind of implies we provide *yet another units package*) and `sys` (which is handy for what we do in there, but could be just imported `as _sys` and not be provided in autocomplete).